### ISSUE TITLE
Steam IA for products available on Steam Store

### ISSUE THREAD

**Issue by stormwindy** [duckduckgo/zeroclickinfo-spice#3298]
It would be nice to have an IA which can give quick information regarding the Steam Store data of the queried game or product, such as price, release date, review stats. Unfortunately Steam has limited API requests (100k per day) but it is stated that it can be increased depending on situation.  I would like to work on this. Before starting, it would be nice to have some input. What are your thoughts in general? 
Steam Web API Terms of Use: https://steamcommunity.com/dev/apiterms
Steam Web API: https://developer.valvesoftware.com/wiki/Steam_Web_API

## People to notify
@stormwindy (Zarnor on Slack)

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

Instant Answer Page: https://duck.co/ia/view/steam_store

**Comment by stormwindy** [comment]
I have been working on Steam Spice. Steam API calls regarding store info uses games' IDs. In order to get those IDs I make an additional call to API which gives me name-id info of games. Should we keep a local copy of game-id data or should we make and additional API call? Keeping a local copy will require constant updates to add newly released games on Steam. On the other hand making an additional call may lower performance. I did not completely tested it yet. I will let you know about the results but I am open to suggestions.

**Comment by nalinbhardwaj** [comment]
Hi @stormwindy, are you still working on this? If not, I can take this up.

Although, I'm not sure how an additional API call is used to find a game's name -> appID relation.

**Comment by pjhampton** [comment]
@nalinbhardwaj You might want to hold off. We're not merging new spices at the moment and not sure when we will resume.

**Comment by pjhampton** [comment]
Hey @stormwindy 

[DuckDuckHack is now in Maintenance Mode](https://duckduckhack.com/) and from now on, we are only accepting issues and PRs for essential bugs and bug fixes.

Unfortunately, with the above in mind, this isn't something we can action and will be closed.

We appreciate you taking the time to contribute and apologize for not being able to triage this issue.
